target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -393,6 +397,9 @@ ath79_setup_macs()
 		lan_mac=$(fritz_tffs -n maca -i $(find_mtd_part "tffs (1)"))
 		wan_mac=$(fritz_tffs -n macb -i $(find_mtd_part "tffs (1)"))
 		;;
+	comfast,cf-e375ac)
+		wan_mac=$(mtd_get_mac_binary art 0x1002)
```

Comment:
As raised in the earlier discussion, I'd actually prefer
$(macaddr_add $(mtd_get_mac_binary art 0x0) 1)
here